wandb/apis/public/api.py

```diff
@@ -1050,28 +1050,38 @@ def list_jobs(self, entity, project):
             return False
 
     @normalize_exceptions
-    def exists(self, name, type=None):
+    def artifact_exists(self, name, type=None):
+        """Return a boolean indicating whether an artifact exists by parsing a path in the form `entity/project/name`.
```

Comment:
Python style is a little strict in this regard, but the summary line should not exceed 80 characters (technically [72](https://peps.python.org/pep-0008/#:~:text=spaces%20for%20indentation.-,Maximum%20Line%20Length,be%20limited%20to%2072%20characters.)... but that's even harder)

it's a good exercise in conciseness though:

```suggestion
        """Return whether an artifact exists.
```

* "a boolean indicating" is unnecessary as it's implied by "whether"
* "by parsing a path in the form ..." is unnecessary as there's a separate section for describing the arguments